### DISCUSSION TITLE
Contribution not displaying on contribution calendar

### DISCUSSION THREAD

**Discussion by rohini-ranjanR** [github/codeql#11406] [Q&A]
Hello,
I have a question On 22nd November I forked a repository and created a forked repository 
![Screenshot 2022-11-24 084522](https://user-images.githubusercontent.com/109434545/203687427-cdae5898-1047-4aed-b826-9845c340eea1.png)

But the problem is that I can see this contribution on my contribution calendar when I logged in with my account

![Screenshot 2022-11-24 084454](https://user-images.githubusercontent.com/109434545/203687581-6cdc8398-1850-4588-ab0b-d2bf38dfeb33.png)

and if I logged out and search for my profile I can't see that contribution in the contribution calendar.

![Screenshot 2022-11-24 084413](https://user-images.githubusercontent.com/109434545/203687684-7264fd06-f42e-4908-968b-fe9178c721c4.png)

When I search for What counts as a contribution (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile) nothing mentioned about forking. Can we say forking can also be counted as a contribution and can be displayed on the Contribution calendar?
If so, what is the issue behind not displaying that contribution in the Contribution calendar when I logged out and search my profile, and displayed it when I logged in?

**Comment by MathiasVP** [comment]
Hi rohini-ranjanR,

Thanks for the issue! This repository is for CodeQL libraries and queries that power GitHub Advanced Security. Thus, I don't think this is the right place for this issue.

If I were you I'd try posting this question in https://github.com/community/community/discussions instead.

**Reply by rohini-ranjanR** [reply]
I post this question https://github.com/community/community/discussions.
Thanks
